.github/workflows/checkbot.yml

```diff
@@ -9,43 +9,41 @@ jobs:
     runs-on: [ubuntu-18.04]
     steps:
       - uses: actions/checkout@v2
-
-      - name: Build test image and publish
+      - name: Build & Publish test image
         run: |
           echo "${{ secrets.DOCKERHUB_PASSWORD }}" | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
-
-          docker build -t dvcorg/cml -f ./docker/Dockerfile .
-          docker build -t dvcorg/cml-py3 -f ./docker/Dockerfile-py3 .
-
-          docker tag dvcorg/cml dvcorg/cml-test
-          docker push dvcorg/cml-test
+          docker build \
+            --build-arg "CML_VERSION=$CML_VERSION" \
+            --build-arg "DVC_VERSION=$DVC_VERSION" \
+            --build-arg "PYTHON_VERSION=$PYTHON_VERSION" \
+            --build-arg "BASE_IMAGE=$BASE_IMAGE" \
+            -t dvcorg/cml-test .
           docker run --name runner --rm \
             -e "REPO_TOKEN=${{ secrets.TEST_GITHUB_TOKEN }}" \
             -e "RUNNER_REPO=https://github.com/${GITHUB_REPOSITORY}" \
             -e "RUNNER_LABELS=cml-docker" \
             -e "RUNNER_IDLE_TIMEOUT=10" \
             dvcorg/cml-test
-
+        env:
+          CML_VERSION: 0
+          DVC_VERSION: 2
+          PYTHON_VERSION: 3.8
+          BASE_IMAGE: 'ubuntu:20.04'
```

Comment:
Before, it was building both py2 & py3, but pushing just py2.

Now it's only building & pushing py3.

Reply:
Taking into account that Py3 is the extended image is OK. The build was to ensure that everything could be built